ivy/functional/ivy/elementwise.py

```diff
@@ -1275,7 +1275,74 @@ def sqrt(
     ret
         an array containing the square root of each element in ``x``. The returned array
         must have a floating-point data type determined by :ref:`type-promotion`.
+    Functional Examples
+    -------------------
+
+    With :code:`ivy.Array` input:
+    
+    >>> x = ivy.array([0, 4, 8])
+    >>> y = ivy.sqrt(x)
+    >>> print(y)
+    ivy.array([0.  , 2.  , 2.83])
+
+    >>> x = ivy.array([1, 2, 4])
+    >>> y = ivy.zeros(3)
+    >>> ivy.sqrt(x, out=y)
+    ivy.array([1.  , 1.41, 2.  ])
+
+    With :code:`ivy.NativeArray` input:
+    
+    >>> x = ivy.native_array([40., 24., 100.])
+    >>> ivy.sqrt(x, out=x)
+    ivy.array([ 6.32,  4.9 , 10.  ])
+
+    >>> x = ivy.native_array([-50., 1000., 34.])
+    >>> y = ivy.sqrt(x)
+    >>> print(y)
+    ivy.array([  nan, 31.6 ,  5.83])
+
+    With :code:`ivy.Container` input:
+    
+    >>> x = ivy.Container(a=ivy.array([44., 56., 169.]), 
+                          b=ivy.array([[49,1], [0,20]]))
+    >>> y = ivy.sqrt(x)
+    >>> print(y)
+    {
+        a: ivy.array([6.63, 7.48, 13.]),
+        b: ivy.array([[7., 1.],
+                      [0., 4.47]])
+    }
+
+    Instance Method Examples
+    ------------------------
+
+    Using :code:`ivy.Array` instance method:
+    
+    >>> x = ivy.array([[1., 2.],  [3., 4.]])
+    >>> y = x.sqrt()
+    >>> print(y)
+    ivy.array([[1.  , 1.41],
+           [1.73, 2.  ]])
 
+    Using :code:`ivy.NativeArray` instance method:
```

Comment:
Hi @Al-ameen007 ,
Sorry for the delay in having your PR reviewed.

Your edits are looking good so far, just a few changes you need to make.

1. Have a look at the description in the link below (from line 209 to 216) and add the same thing into your `sqrt` docstring.

https://github.com/unifyai/ivy/blob/7b741439dc15db949a27d6deb530c3f17dc49e41/ivy/functional/ivy/elementwise.py#L209
(Also remember to update the URL to the correct one for `sqrt`)

2. There's no need to include a `ivy.NativeArray`  instance method since the output will vary depending on which backend is being used, so you can remove the example at line 1327.